<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinaryFileBuilder</class>
 <widget class="QWidget" name="BinaryFileBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>193</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="uiMainLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="uiInputFileLabel">
     <property name="text">
      <string>&amp;Input file:</string>
     </property>
     <property name="buddy">
      <cstring>uiInputFileEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="uiOutputNameLabel">
     <property name="text">
      <string>&amp;Output name:</string>
     </property>
     <property name="buddy">
      <cstring>uiOutputNameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="uiOutputNameEdit"/>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="uiInputFileLayout">
     <item>
      <widget class="QLineEdit" name="uiInputFileEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="uiInputBrowseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="uiVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
